### PR TITLE
Add a species props not required and render table

### PR DESCRIPTION
**Pull request by lingyun1010**

### CODE REVIEW

File: src/ExperimentTable.js [hunk 1]
```diff
@@ -95,6 +95,13 @@ class ExperimentTable extends React.Component {
     })
   }
 
+  componentDidMount(){
```
Comment (alfonsomunozpomer) [inline src/ExperimentTable.js]: You don’t need to add `componentDidMount`. Just initialise the state fields `searchQuery` and `searchedColumnIndex` and the props will naturally flow down to `TableContent`.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
A minor change. Only use lifecycle methods when there’s no other alternative.

File: src/ExperimentTable.js (hunk 1 above)
Comment (lingyun1010) [reply src/ExperimentTable.js]: `columnNumber` is not a stateful variable. We need to specify which column the `species` value is searching.

Comment (alfonsomunozpomer) [reply src/ExperimentTable.js]: But `searchedColumnIndex` is.

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Much better!

I would initialise the constructor with:
```javascript
searchQuery: props.species.trim(),
searchedColumnIndex: props.species.trim() && ...
```

So the logic is consistent across the component. Also, fix the tests which are currently failing (remember that if a prop isn’t required you should provide a default value).

File: src/ExperimentTable.js [hunk 2]
```diff
@@ -11,9 +11,10 @@ class ExperimentTable extends React.Component {
     super(props)
     this.entriesPerPageOptions = [10, 25, 50]
     this.state = {
-      searchQuery: this.props.species,
+      searchQuery: this.props.species.trim(),
       orderedColumnIndex: 0,
-      searchedColumnIndex: this.props.tableHeader.findIndex(header => header.dataParam === "species"),
+      searchedColumnIndex: this.props.species.trim() ?
+        this.props.tableHeader.findIndex(header => header.dataParam === "species") : 1,
```
Comment (alfonsomunozpomer) [inline src/ExperimentTable.js]: Use backtick for strings, this should be flagged by eslint.

**Review by alfonsomunozpomer — APPROVED** [review]
Change the string and everything’s good to go!

**Pull request merged**